frontend/src/static/js/v_authorship.js

```diff
@@ -13,7 +13,7 @@ function toggleNext(ele) {
   }
 
   parent.className = classes.join(' ');
-};
+}
```

Comment:
for lines that are irrelevant to your PR, please avoid modifying them as it may cause unnecessary conflict and also deviates from the purpose of your PR.

Reply:
This comes from ESLint